trestle/core/control_io.py

```diff
@@ -1048,7 +1089,11 @@ def read_new_alters(control_path: pathlib.Path) -> List[prof.Alter]:
                     new_alters.append(alter)
             else:
                 ii += 1
-        return new_alters
+        param_content = header.get(const.SET_PARAMS_TAG, {})
```

Comment:
I believe this needs to default to none.

Reply:
When being written out.

Reply:
I believe these are all addressed but I can't replicate the override of a parameter that was set to ''.  Please confirm and describe the test.